### DISCUSSION TITLE
Select a column with table name?

### DISCUSSION THREAD

**Discussion by orangesoup** [SeaQL/sea-query#736] [Q&A]
Hey! I'm coming from the Go world and been using https://github.com/Masterminds/squirrel there.

I could do something like this there:

```go
Columns("table.created_at", ...)
```

How can I do something similar here?

When I do 

```rust
.columns(Alias::new("table.created_at"), ...)
```

the generated SQL is `'table.created_at'` (with `, markdown things) which is obviously not going to work.

Also, since I'm only using a query builder part of sea-query, is there a way to use simple strings, instead of using Alias::new("...") everywhere?

Cheers!

**Comment by jvliwanag** [comment]
You'd want to pass in a tuple. So something like:

```
.column((Alias::new("table"), Alias::new("created_at"))
```
